### ISSUE TITLE
Make app deployable for local development

### ISSUE THREAD

**Issue by cecilia-donnelly** [redcross/arcdata#73]
As alluded to in #67, people interested in contributing changes and doing local testing need to be able to get this app running locally.  Currently, thanks to the work by @benubois in PR #72 and by @mchu0010 [here](https://github.com/OpenTechStrategies/arcdata/tree/deployment_experiments), pieces of the app can be deployed locally, but not the entire product.

This issue depends on #67, and I'll open more specific issues for certain problems with deployment.  Consider this the umbrella issue to refer to for everything to do with making DCSOps fully deployable locally.

**Comment by cecilia-donnelly** [comment]
As of e1ce92b, running `db:seed` (or the setup script in the dev-setup branch) creates a "regular" user and an admin user.  The admin user has permission to create incidents.  It looks like admins are assigned through a few steps: roles are created, positions are created (we already have several of these in the sample data), and then users are assigned as members of these role/position groups.

I think that the positions step is meant to make the assignment of privileges more readable, or maybe it has implications for the UI.  The positions are more like "CNI DAT Administrator" while the roles are things like "incidents_admin."  Presumably the positions will each have known roles / permissions assigned to them, and new users can then be assigned to given positions.

I haven't seen any user administration in the UI yet.

**Comment by cecilia-donnelly** [comment]
We don't have a full list of user stories, but the admin user (dispatcher) I've created locally can now:
- create a new incident
- close an incident
- assign responders to an incident
- create chapters, counties, roles, etc. from the app

More to come!
The dispatcher should also be able to:
- view responder history
- create shifts
- create new volunteer users in the app

The dispatch and volunteer users should both be able to:
- sign up for shifts (I believe they can now, but I will test more)
- anything else?

I also see a lot of options in the admin menu that are active but I don't yet know what to do with (e.g. Deployments, Dispatch Logs).  I know based on some of the other issues in this repository that dispatchers can communicate with responders via SMS, and I think some of that is managed through DSCOps, but I haven't tested that at all.

**Comment by cecilia-donnelly** [comment]
@lorenanicole, did you see this issue?  I've been working in the [dev-setup](https://github.com/redcross/arcdata/commits/dev-setup) branch to get a local instance of this up and running.

**Comment by cecilia-donnelly** [comment]
While looking at issue #70, I noticed that my test admin user doesn't have access to paths of the form `/incidents/_CHAPTER_ABBREVIATION_/incidents/_INCIDENT_NUMBER_/responders`.  This line in app/controllers/incidents/responders_controller.rb is blocking access:

`load_and_authorize_resource class: 'Incidents::ResponderAssignment'`

The `load_and_authorize_resource` function comes from the CanCan library, which we're evidently using [a fork](https://github.com/jlaxson/cancan) of.  Clearly this line is asking whether the logged-in user is authorized on that class, but I'm not yet sure how to authorize a user for the class.  I thought this user had all necessary roles, but maybe not?  Looking into it.

**Comment by kfogel** [comment]
I re-opened this, and removed the `wontfix` label, but added `duplicate` because I think this work is sort of implied in issue #118 anyway, and may in any case be partly resolved by the recent addition of the `INSTALL.md` file.  I'm undecided about whether to re-open issue #67; we might want to.
